src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ReconfigureFundingCycleSettingsPage/hooks/useReconfigureFundingCycle.ts

```diff
@@ -65,111 +65,109 @@ export const useReconfigureFundingCycle = ({
   const reconfigureV2V3FundingCycleWithNftsTx =
     useReconfigureV2V3FundingCycleWithNftsTx()
 
-  const {
```

Comment:
The diff went whack here but the only thing that's changed is:
- Flesh out the `editingFundingCycleConfig` object inside `reconfigureFundingCycle()` rather than outside
- Add optional `latestEditingData?: EditingFundingCycleConfig` to `reconfigureFundingCycle()`
- Dependencies changed

Reply:
got it, ty!